conjur/cli.py

```diff
@@ -19,21 +19,30 @@
 import requests
 
 # Internals
-from conjur.argparse_wrapper import ArgparseWrapper
-from conjur.client import Client
+from conjur.wrappers.argparse_wrapper import ArgparseWrapper
+from conjur.api.client import Client
 from conjur.constants import DEFAULT_NETRC_FILE, DEFAULT_CONFIG_FILE
-from conjur.credentials_data import CredentialsData
-from conjur.credentials_from_file import CredentialsFromFile
-from conjur.host import HostController
-from conjur.host.host_resource_data import HostResourceData
-from conjur.init import ConjurrcData
-from conjur.list import ListData, ListController
-from conjur.list.list_logic import ListLogic
-from conjur.login import LoginLogic, LoginController
-from conjur.logout import LogoutController, LogoutLogic
-from conjur.policy import PolicyData, PolicyLogic, PolicyController
-from conjur.variable import VariableLogic, VariableController, VariableData
-from conjur.user import UserController, UserInputData, UserLogic
+from conjur.controller.host_controller import HostController
+from conjur.controller.list_controller import ListController
+from conjur.controller.login_controller import LoginController
+from conjur.controller.logout_controller import LogoutController
+from conjur.controller.policy_controller import PolicyController
+from conjur.controller.user_controller import UserController
+from conjur.controller.variable_controller import VariableController
+from conjur.logic.credentials_from_file import CredentialsFromFile
+from conjur.data_object.conjurrc_data import ConjurrcData
+from conjur.data_object.credentials_data import CredentialsData
+from conjur.data_object.host_resource_data import HostResourceData
+from conjur.data_object.list_data import ListData
+from conjur.data_object.policy_data import PolicyData
+from conjur.data_object.user_input_data import UserInputData
+from conjur.data_object.variable_data import VariableData
+from conjur.logic.list_logic import ListLogic
+from conjur.logic.login_logic import LoginLogic
+from conjur.logic.logout_logic import LogoutLogic
+from conjur.logic.policy_logic import PolicyLogic
+from conjur.logic.user_logic import UserLogic
+from conjur.logic.variable_logic import VariableLogic
 from conjur.version import __version__
 
 # pylint: disable=too-many-statements
```

Comment:
all new file directory __init__ should not be empty IMO. They should contain the controllers that they represent from example `import init_controller from InitController`. 

Please also add a docstring explaining what each directory contains

"""
Controller module

This metafile includes all controllers required in this project
"""

same for data_object and same for api, etc etc

Reply:
Added